verificarlo.in.in

```diff
@@ -44,61 +44,66 @@ clang = '@CLANG_PATH@'
 opt = llvm_bindir + '/opt'
 dragonegg = "@DRAGONEGG_PATH@"
 gcc = "@GCC_PATH@"
-FORTRAN_EXTENSIONS=[".f", ".f90", ".f77"]
-C_EXTENSIONS=[".c", ".cpp", ".cc", ".cxx"]
+FORTRAN_EXTENSIONS = [".f", ".f90", ".f77"]
+C_EXTENSIONS = [".c", ".cpp", ".cc", ".cxx"]
+
 
 class NoPrefixParser(argparse.ArgumentParser):
     # ignore prefix autocompletion of options
     def _get_option_tuples(self, option_string):
         return []
 
+
 def fail(msg):
     print(sys.argv[0] + ': ' + msg, file=sys.stderr)
     sys.exit(1)
 
+
 def is_fortran(name):
     return os.path.splitext(name)[1].lower() in FORTRAN_EXTENSIONS
 
+
 def is_c(name):
     return os.path.splitext(name)[1].lower() in C_EXTENSIONS
 
+
 def parse_extra_args(args):
     sources = []
     options = []
 
     for a in args:
         if is_fortran(a):
-	    if not dragonegg:
-		fail("fortran not supported. "
-		     + "--without-fortran was used during configuration.")
-	    sources.append(a)
-	elif is_c(a):
-            sources.append(a)
+            if not dragonegg:
+                fail("fortran not supported. "
+                     + "--without-fortran was used during configuration.")
+                sources.append(a)
+            elif is_c(a):
+                sources.append(a)
         else:
             options.append(a)
 
     return sources, ' '.join(options)
 
-def fail(msg):
-    print(sys.argv[0] + ': ' + msg, file=sys.stderr)
-    sys.exit(1)
 
 def shell(cmd):
     try:
         subprocess.check_call(cmd, shell=True)
     except subprocess.CalledProcessError:
         fail('command failed:\n' + cmd)
 
+
 def linker_mode(sources, options, output, args):
-    shell('{clang} -c -O2 -static -o .vfcwrapper.o {vfcwrapper} -I {mcalib_includes}'.format(
+    positionmode = "static" if args.static else "fPIC"
+    shell('{clang} -c -O2 -{positionmode} -o .vfcwrapper.o {vfcwrapper} -I {mcalib_includes}'.format(
```

Comment:
can we include the "-" inside positionmode, just to be consistent with what we are doing for the other options.

Reply:
Done - thanks!